src/app/lib/utilities/getRemoteConfig/index.js

```diff
@@ -0,0 +1,45 @@
+import toggles from '#lib/config/toggles';
+import constructTogglesEndpoint from '#contexts/ToggleContext/utils/constructTogglesEndpoint';
+import nodeLogger from '#lib/logger.node';
+import { CONFIG_REQUEST_RECEIVED, CONFIG_FETCH_ERROR } from '#lib/logger.const';
+import getOriginContext from '#contexts/RequestContext/getOriginContext';
+
+const logger = nodeLogger(__filename);
+
+const getRemoteConfig = async (service, cache) => {
+  const environment = process.env.SIMORGH_APP_ENV || 'local';
```

Comment:
What's the need for the `||`, is it because we don't always expect `process.env.SIMORGH_APP_ENV` to be set?